### PR TITLE
Update tesla.markdown

### PR DESCRIPTION
**Pull request by NicklasMD**
Added "Note: MFA on your Tesla Account is not supported at this time."

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @NicklasMD, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17353).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/tesla.markdown [hunk 1]
```diff
@@ -28,6 +28,8 @@ ha_platforms:
 
 The `Tesla` integration offers integration with the [Tesla](https://auth.tesla.com/login) cloud service and provides presence detection as well as sensors such as charger state and temperature.
 
+Note: MFA on your Tesla Account is not supported at this time.
```
Comment (frenck) [inline source/_integrations/tesla.markdown]: - I think this should be in the configuration section (not in the introduction text).
- Format this as a note, see: <https://developers.home-assistant.io/docs/documenting/create-page#html>

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @NicklasMD 👍

File: source/_integrations/tesla.markdown (hunk 1 above)
Comment (NicklasMD) [reply source/_integrations/tesla.markdown]: I've changed the suggested edits.

**Pull request merged**